plugins/kubernetes/app/models/watchers/pod_watcher.rb

```diff
@@ -0,0 +1,84 @@
+module Watchers
+  class PodWatcher < BaseWatcher
+    POD_PHASES = %w[Pending Running Succeeded Failed Unknown]
+    
+    attr_reader :pod_info
+
+    def initialize(client, namespace, name: nil, label_selector: nil, log: true)
+      raise ArgumentError.new('Must specify either pod_name or label_selector') if name.blank? && label_selector.blank?
```

Comment:
One of `name` or `label_selector` needs to be defined.  Using keyword arguments like that would throw an error if either of them were nil, which isn't what I want.